src/app/admin/components/ReportsList.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+import { Button } from '@/components/ui/button';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { Flag } from 'lucide-react';
+import type { AdminProfile } from './ProjectSubmissions';
+
+export interface CandidateProfile {
+    userId: string;
+    displayName: string;
+    location: string | null;
+    language: string | null;
+    resumeURL: string | null;
+    bio: string | null;
+    experience: string | null;
+    portfolioURL: string | null;
+    linkedinURL: string | null;
+    imageURL: string | null;
+}
+
+export interface User {
+    id: string;
+    name: string | null;
+    email: string;
+    image: string | null;
+    role: 'candidate' | 'admin' | 'recruiter' | null;
+    createdAt: Date | null;
+    updatedAt: Date | null;
+}
+
+export interface Report {
+    id: string;
+    candidateId: string | null;
+    reporterId: string | null;
+    reason: string;
+    additionalDetails: string;
+    createdAt: Date | null;
+    action: 'banned' | 'archived' | null;
+    actionedAt: Date | null;
+    actionedBy: string | null;
+    bannedUserDisplayName: string;
+    candidateProfile: CandidateProfile | null;
+    reporter: User | null;
+    actioner: AdminProfile | null;
+}
```

Comment:
Use RouterOutputs here too
```suggestion
type Report = RouterOutputs['reports']['getAll'][number];
```